### PR TITLE
UIIN-2399: Instance 3rd pane header: Add Local/Shared indication

### PR DESCRIPTION
**Pull request by mariia-aloshyna**
## Purpose

In a consortium environment, Instance records may be shared amongst the member institutions and users need a way to know if the record they are viewing/editing is either shared or local to their institution.

## Approach

- Check whether a user is in the central tenant/member library tenant to define shadow/local instance
- Render separate instance title for instances in consortia mode

### For non-consortia users
image

### Member library tenant - shadow instance
image

### Member library tenant - local instance
image

### Central tenant
image

### CODE REVIEW

**Comment by Dmitriy-Litvinenko** [comment]
Hello @mariia-aloshyna Could you please check broken tests?
```
Test Suites: 1 failed, 214 passed, 215 total
Tests:       27 failed, 726 passed, 753 total
```

**Comment by mariia-aloshyna** [comment]
> Hello @mariia-aloshyna Could you please check broken tests?
> 
> ```
> Test Suites: 1 failed, 214 passed, 215 total
> Tests:       27 failed, 726 passed, 753 total
> ```

Hi @Dmitriy-Litvinenko it's all fixed now, we had these failures because https://github.com/folio-org/ui-inventory/pull/2215 hasn't been merged yet.

File: src/ViewInstance.js [hunk 1]
```diff
@@ -734,6 +738,24 @@ class ViewInstance extends React.Component {
     );
   };
 
+  renderPaneTitle = (instance) => {
+    const { stripes } = this.props;
+
+    const isInstanceShared = checkIfUserInCentralTenant(stripes)
+      || (checkIfUserInMemberTenant(stripes) && instance?.source.startsWith(CONSORTIUM_PREFIX));
```
Comment (Dmytro-Melnyshyn) [inline src/ViewInstance.js:745]: Please use just merged [checkIfSharedInstance](https://github.com/folio-org/ui-inventory/blob/master/src/utils.js#L783).

File: test/jest/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    ['@babel/preset-react', { runtime: 'automatic' }],
+  ],
+  plugins: [
+    ['@babel/plugin-proposal-decorators', { legacy: true }],
+    ['@babel/plugin-transform-class-properties', { loose: true }],
+    ['@babel/plugin-transform-private-methods', { loose: true }],
+    ['@babel/plugin-transform-private-property-in-object', { loose: true }],
+    '@babel/plugin-transform-runtime',
+  ],
+};
```
Comment (Dmytro-Melnyshyn) [inline test/jest/babel.config.js:13]: Tests are no longer failing, this can be removed.

File: test/jest/jest-transformer.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+const babelJest = require('babel-jest');
+
+const babelConf = require('./babel.config');
+
+module.exports = babelJest.createTransformer(babelConf);
```
Comment (Dmytro-Melnyshyn) [inline test/jest/jest-transformer.js:5]: The same here.

File: package.json [hunk 4]
```diff
@@ -864,12 +864,14 @@
...
     "@folio/stripes-util": "^6.0.0",
     "@formatjs/cli": "^4.2.20",
+    "babel-jest": "^29.6.1",
     "core-js": "^3.6.4",
     "eslint": "^7.13.0",
     "eslint-plugin-filenames": "^1.3.2",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jest": "^24.0.0",
     "inflected": "^2.0.4",
+    "jest": "^29.6.1",
```
Comment (Dmytro-Melnyshyn) [inline package.json:874]: Please remove added packages, tests are no longer failing.

File: src/ViewInstance.test.js [hunk 5]
```diff
@@ -208,7 +210,71 @@ describe('ViewInstance', () => {
...
+
+          expect(getByText(expectedTitle)).toBeInTheDocument();
+        });
+      });
+    });
+  });
   describe('Action Menu', () => {
+    beforeAll(() => {
+      StripesConnectedInstance.prototype.instance.mockImplementation(() => instance);
+    });
```
Comment (Dmytro-Melnyshyn) [inline src/ViewInstance.test.js:276]: After merging the master this will become unnecessary.

**Review by Dmytro-Melnyshyn — APPROVED** [review]

**Review by artem-blazhko — APPROVED** [review]

**Pull request merged**